__main__.py

```diff
@@ -0,0 +1 @@
+/home/pablo/Documents/mathicsscript/mathicsscript/__main__.py
```

Comment:
Was this added inadvertently?

Reply:
Yes, I didn't intent to add this (I don't event know what it is).